sdk/keyvault/azure-security-keyvault-certificates/samples/certificate-ImportCertificate/certificate-ImportCertificate.cpp

```diff
@@ -55,8 +55,9 @@ int main()
       options.Policy.KeySize = 2048;
       options.Policy.ContentType = CertificateContentType::Pem;
       options.Policy.Exportable = true;
+      options.Properties.Name = pemName;
```

Comment:
name can't be part of options.  `options` represent non-required arguments that can be defaulted.

Reply:
If we make options a required parameter for the API, then we have to expose a ctor for options that takes the required params ( like the name ), so we prevent sending an invalid request by trying to use a default-created options

Reply:
that's what the archboard wants , match .net which has name in the options

Reply:
But in that case, do we want one specific constructor for the options which takes the name as argument?  So we are sure that options can't be created without a name?  

Otherwise, we are opening the door for simple bugs of doing client.StartCreateCertificate( { } );  // This would call the default options constructor and init an empty name..  That's not what we want, do we?